pom.xml

```diff
@@ -43,6 +43,11 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-data-mongodb</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>com.graphql-java-kickstart</groupId>
+			<artifactId>playground-spring-boot-starter</artifactId>
```

Comment:
just curious what this is?

Reply:
ooo it means we can now interact with the api in one of those playground things, just go to localhost:8080/playground - i've updated the readme with this info